xapian-core/tests/harness/backendmanager.h

```diff
@@ -139,6 +139,13 @@ class BackendManager {
     /// Get a remote database instance with the specified timeout.
     virtual Xapian::Database get_remote_database(const std::vector<std::string> & files, unsigned int timeout);
 
+     /*
+     * Get the args for opening a writable remote database with the
+     * specified timeout.
+     */
+    virtual std::string get_writable_database_args(const std::string& path,
```

Comment:
The first line of the comment here is indented by an extra space, so the `*` doesn't align with the following lines.

But also this really should be a doxygen documentation comment which (for a multiline comment) start with `/**`, with the brief description following on the same line.